### ISSUE TITLE
Can't add new account

### ISSUE THREAD

**Issue by mateuszprze** [farshadz1997/Microsoft-Rewards-bot-GUI-V2#32]
Hello,

there is error. When I open main and go to Account - I cant add new one (Account file not exist, Open account first then try to add account). I tried open file accounts.json and change ther, but still this same. Have no idea what to do to add my account.

**Comment by farshadz1997** [comment]
Hello,
there's a sample account in accounts.json, isn't there? edit it with your credentials and add more account if you want.
If you think there's problem with accounts.json try to delete it and restart application, it will create another one

**Comment by mateuszprze** [comment]
Yes i Did exacly like that. I edit it with my email and password, saved restart app and nothing appear inside app

**Comment by farshadz1997** [comment]
open accounts.json with notepad or other text editor and look for your account in it whether is saved or not.

**Comment by mateuszprze** [comment]
That my code inside

[
    {
        "username": "m********@outlook.com",
        "password": "M****!"
    }
]

**Comment by farshadz1997** [comment]
your account should be appeared in app. there's no problem with it

**Comment by mateuszprze** [comment]
It should works in Win 10 ?

**Comment by farshadz1997** [comment]
yes

**Comment by Colin-Cai0318** [comment]
How do I add multiple accounts？Like this？

**Comment by Colin-Cai0318** [comment]
How do I add multiple accounts？like this？
[
    {
        "username": "username1",
        "password": "password1"
    },
    {
        "username": "username2",
        "password": "username2",
    }
]
